tensorflow_app/views/export_graphdef.py

```diff
@@ -1,20 +1,20 @@
 import os
 import string
 import random
+import tensorflow as tf
 from django.views.decorators.csrf import csrf_exempt
 from django.http import JsonResponse
 from keras_app.views.export_json import export_json
+from tensorflow.keras.backend import clear_session
```

Comment:
```suggestion
from keras.backend import clear_session
```
Looks like travis build is failing when we try to access `tensorflow.keras.backend`. Can you try this instead? If this suggestion also fails travis build, we can try using `from keras import backend as K` and then call `K.clear_session()` instead. Try making this change at every instance where you used `keras.backend`.